handlers.go

```diff
@@ -278,13 +278,12 @@ func (s *Server) ServeAPI() error {
 	admin.POST("/cm/repinall/:shuttle", s.handleShuttleRepinAll)
 
 	//	peering
-	adminPeering := admin.Group("/peering")
-	adminPeering.POST("/peers", s.handlePeeringPeersAdd)
-	adminPeering.DELETE("/peers", s.handlePeeringPeersRemove)
-	adminPeering.GET("/peers", s.handlePeeringPeersList)
-	adminPeering.POST("/start", s.handlePeeringStart)
-	adminPeering.POST("/stop", s.handlePeeringStop)
-	adminPeering.GET("/status", s.handlePeeringStatus)
+	admin.POST("/peering/peers", s.handlePeeringPeersAdd)
```

Comment:
Is there a reason why we needed to remove the group here? It make sense to utilize the grouping in this case since we have multiple functions for a `peering`.